FoodFite/Models/UserProfile.cs

```diff
@@ -13,11 +13,21 @@ namespace FoodFite.Models
     public class UserProfile
     {
         public string Name { get; set; }
+        public string Opponent { get; set; }
```

Comment:
Why is Opponent and Weapon here?

Reply:
it's need right now to just get the bot running. Those are values that need to persist. We need to leave this here until we find a better way to persist it.

Reply:
ok